manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "admin-pages",
-  "version": "4.54.1",
+  "version": "4.55.0",
```

Comment:
No need to update the version number here, the CI will take care of that once the PR is merged :)

Reply:
```suggestion
  "version": "4.54.1",
```